app/views/duty_deferment_account/duty_deferment_account.scala.html

```diff
@@ -35,7 +35,7 @@
             @statement.historicEoriHeading.map(identity)
             @statement.ddHeadWithEntriesOrNoStatements.ddHeadWithEntry.map { headWithEntry =>
                     @headWithEntry.ddHead
-                    @headWithEntry.entries.map(identity)
+                    @headWithEntry.entries
```

Comment:
Could you please add a test case (for view) to check the count of 'Show all sections' text ?